src/routes/v2.js

```diff
@@ -0,0 +1,56 @@
+'use strict';
+
+const express = require('express');
+const dataModules = require('../models');
+const bearer = require('../auth/middleware/bearer');
+const acl = require('../auth/middleware/acl');
+const router = express.Router();                            
+
+router.param('model', (req, res, next) => {
+  const modelName = req.params.model;
+  if (dataModules[modelName]) {
+    req.model = dataModules[modelName];
+    next();
+  } else {
+    next('Invalid Model');
+  }
+});
+
+router.get('/:model', acl, handleGetAll);
+router.get('/:model/:id', acl, handleGetOne);
+router.post('/:model', bearer, acl, handleCreate);
+router.put('/:model/:id', bearer, acl, handleUpdate);
+router.delete('/:model/:id', bearer, acl, handleDelete);
```

Comment:
All routes should have bearer auth.

Additionally, the acl usage is not being done correctly. If you look at the function in acl.js, it is currying a middleware function. That means we need to actually invoke the function with the capability we want to test for:
```js
router.delete('/:model/:id', bearer, acl('delete'), handleDelete);